src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/Model.cs

```diff
@@ -79,6 +79,14 @@ public class Node : AbstractNode
         public string? ChildrenOrder;
 
         public override bool IsAbstract => false;
+
+        [XmlAttribute(AttributeName = "HasType")]
+        public string HasTypeText;
+        public bool HasType => HasTypeText == "true";
+
+        [XmlAttribute(AttributeName = "HasConstantValue")]
```

Comment:
>HasConstantValue [](start = 39, length = 16)

I don't recall whether a contributor added an XSD file for IOperation xml file. If we have one, we should update it too

Reply:
We do not as far as I know.